snapshot-labs/sx-ui
# feat: add Snapshot Hub integration as ready only platform

### Summary

Closes: https://github.com/snapshot-labs/pitches/issues/52

### How to test

1. Go to http://localhost:8080/#/s:arbitrumfoundation.eth
2. It loads.
3. Results are visible for basic voting proposals.
4. Check results for non-basic voting proposal: http://localhost:8080/#/s:arbitrumfoundation.eth/proposal/0xf22530295daee96dffd7f70854475c06216a4d3594929672f71c12bf638bb0c8
5. Check results for basic voting proposal: http://localhost:8080/#/s:arbitrumfoundation.eth/proposal/0x76ff3c3e1750cfbe34dd62ed22c0c126168963d6acce2217acc4bcc7e9e623b7

### Screenshots

<img width="1045" alt="image" src="https://github.com/snapshot-labs/sx-ui/assets/1968722/b1afb400-690d-40f0-9619-3aefd2f58a55">
<img width="1044" alt="image" src="https://github.com/snapshot-labs/sx-ui/assets/1968722/0bee3a08-83cb-47ec-931a-b886b137560a">
<img width="1047" alt="image" src="https://github.com/snapshot-labs/sx-ui/assets/1968722/919ef348-f718-46d2-94de-0ff158619235">
